### ISSUE TITLE
Add drop_last to Dataloader

### ISSUE THREAD

**Issue by MaxJoas** [jan-forest/autoencodix_package#79]
## Context
When either batch_size =1 (which is prevented by Pydantic Config) or n_samples % batch_size = 1, we get an error when calculating BatchNorm in our models.

## Fix
add drop_last to DataLoader
